flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/ExecutionSlotAllocator.java

```diff
@@ -29,10 +29,10 @@
     /**
      * Allocate slots for the given executions.
      *
-     * @param executionVertexSchedulingRequirements The requirements for scheduling the executions.
+     * @param executionVertexIds The requirements for scheduling the executions.
```

Comment:
"The requirements for scheduling the executions" -> "Execution vertices to allocate slots for"